pypeit/core/findobj_skymask.py

```diff
@@ -66,57 +66,68 @@ def create_skymask(sobjs, thismask, slit_left, slit_righ, box_rad_pix=None, trim
         global sky subtraction (True means the pixel is usable for sky
         subtraction, False means it should be masked when subtracting sky).
     """
+    # Number of objects
     nobj = len(sobjs)
-    ximg, _ = pixels.ximg_and_edgemask(slit_left, slit_righ, thismask, trim_edg=trim_edg)
-    # How many pixels wide is the slit at each Y?
-    xsize = slit_righ - slit_left
-    #nsamp = np.ceil(np.median(xsize)) # JFH Changed 07-07-19
-    nsamp = np.ceil(xsize.max())
+    if nobj == 0:
```

Comment:
doesn't look like numpy issues, so I assume some other
bits and pieces slipped in.  which is fine